packages/components/alert/src/alert.ts

```diff
@@ -1,8 +1,9 @@
 import { TypeComponentsMap } from '@element-plus/utils/icon'
 import { buildProps, keyOf } from '@element-plus/utils/props'
 import type { ExtractPropTypes } from 'vue'
+import type Alert from './alert.vue'
 
-export type AlertEffect = 'light' | 'dark'
+export const alertEffect = ['light', 'dark'] as const
```

Comment:
This is for consistency with prop name.

Just like https://github.com/element-plus/element-plus/blob/38432930344a040812edeb6c38f6a82fc018e336/packages/components/button/src/button.ts#L6

Reply:
I think we need to distinguish from types and variables since this is a variable which represents the acceptable value for the alert effect, we should keep it that way, besides your API field was named `values`, it'd be much more reasonable for this variable to be called in **alertEffects**

Note that there are difference between **alertEffects** and **AlertEffect**

Reply:
Updated.